docs/source/tools/visual.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Contract Visulization
+#########
+
+Introduction
+************
+
+You can generate visual graphs for DAML contracts visual command via command line tool. This program generates dot file which in turn can fed in to graphviz engine to produce an image.
```

Comment:
This is a bit ambiguous as to what the user actively needs to do. And I think you can  merge this with the instructions below. I'd suggest something like: 
 
```suggestion
You can generate visual graphs for the contracts in your DAML project. To do this:

 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
 4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
```

Reply:
Cool adapted the changes. This looks a lot easier.